src/mon/MonmapMonitor.cc

```diff
@@ -129,7 +129,9 @@ void MonmapMonitor::create_pending()
   pending_map = *mon.monmap;
   pending_map.epoch++;
   pending_map.last_changed = ceph_clock_now();
-  dout(10) << __func__ << " monmap epoch " << pending_map.epoch << dendl;
+  pending_map.removed_ranks.clear();
+  dout(10) << __func__ << " monmap epoch " << pending_map.epoch 
+    << " removed_ranks " << pending_map.removed_ranks << dendl;
```

Comment:
...but this log line probably needs to go before clearing the removed_ranks? I'm not quite sure what the point of this change is, unless you were debugging to discover the removed_ranks were still present?